Automattic/newspack-plugin
# hotfix: disable author filter for co-authors

### All Submissions:






### Changes proposed in this Pull Request:



Disable Authors filter if Co-Authos Plus plugin is enabled

### How to test the changes in this Pull Request:

1. Verify Posts page and see the authors filter
2. Activate CAP
3. See the filter is gone

### Other information:
